alerts/cluster-network-operator/NoRunningOvnMaster.md

```diff
@@ -0,0 +1,75 @@
+# NoRunningOvnMaster
+
+## Meaning
+
+This [alert][NoRunningOvnMaster] is triggered when there are no
+OVN-Kubernetes master control plane pods
+[Running][PodRunning]
+This is a critical-level alert if no OVN-Kubernetes master control plane pods
+are not running for `10m`.
```

Comment:
Does 10m seem too high without a running master? I think this might be a more severe condition than other critical alerts of OVN so could make it smaller perhaps to alert it sooner? My reference here is with [etcdNoLeader](https://github.com/openshift/cluster-etcd-operator/blob/master/manifests/0000_90_etcd-operator_03_prometheusrule.yaml#L32-L42) alert which would be fired after 1m. We might want to keep it to be 1-3 minutes. Let me know your thoughts on it.

Reply:
Oh ok there's a separate `NoOvnMasterLeader` alert too. And I missed that the CNO would go degraded here and not down so perhaps 10m would be ok too.

Reply:
Hi @ravitri There is another review[1] for those alerts, like when it should be triggered. And Martin suggested to reduce this from 10m to 1m.
One concern of mine, as if we are setting the trigger hold time too short. It might be mis-fired when the master node reboot takes longer. WDYT?

[1]https://github.com/orgs/openshift/teams/sre-alert-sme/discussions/2

Reply:
This alert will only fire if _all_ the OVN-K master pods are not running. 1m is probably too short an interval and maybe 10m is too large, but until we have data for a correct time, we can leave it for now.
Please voice your opinion on this in the google doc in the link bo attached.

Reply:
Yeah, agreed that 1m is too short too.. I was thinking about 3 minutes but then I agree with Martin. We need more data to decide better so can revisit it in future. Resolving this comment :+1: